main/glossary/README.md

```diff
@@ -8,24 +8,34 @@ not be trusted as an accurate depiction, since it is provided by
 the maker of the mint and could be deceptive, but is useful for debugging and double-checking.
 
 The AllegedName must be a string.
-
+<a name="AmountMath"></a>
```

Comment:
Are the HTML anchors redundant with the header lines? Doesn't MarkDown create anchors for every header?